src/apps/inventory/views.py

```diff
@@ -18,9 +18,15 @@ class ProductDetailView(DetailView):
     def get_queryset(self):
         category_slug = self.kwargs.get("category_slug")
         category = get_object_or_404(Category, slug=category_slug)
-        queryset = Product.objects.filter(category=category, is_active=True, stock__gt=0)
```

Comment:
Why deleting stock greater then 0 are we suppose to retrive stuff that are out of stock?